### PR TITLE
test: add test case for unconvertible grch38 variant (issue #196)

### LINKED ISSUE #196: test: add test case for a variant in GRCh38 that can't be converted to GRCh37
### Feature description

We implemented a number of test cases for issue #147 (lifting over variants on registration). However, I was unable to find an example of a variant in GRCh38 that could not be converted to GRCh37. In order to have full testing coverage, we should find such a variant and add it to our test cases.

### Use case

Our tests should cover all conceivable edge cases, so that our code remains robust.

### Acceptance Criteria

1. The `unconvertible_grch38_variant` entry in `tests/data/liftover_variants.py` is filled in with a JSON representation of a GRCh38 VRS variant that can't be converted to GRCh37. 
2. In `tests/test_liftover.py`, the `FAILURE_CASES` constant should have the "unconvertible_grch38_variant" entry un-commmented out.

### Proposed solution

_No response_

### Alternatives considered

_No response_

### Implementation details

_No response_

### Potential Impact

_No response_

### Additional context

_No response_

### Contribution

None

### PR DESCRIPTION
**Pull request by jennifer-bowser**
closes #196

### CODE REVIEW

File: tests/data/liftover_variants.py [hunk 1]
```diff
@@ -225,7 +225,28 @@
         "expected_output": CoordinateConversionError,
     },
     "unconvertible_grch38_variant": {
-        "variant_input": {},  # TODO: Add an actual test case for this. See Issue #196.
+        "variant_input": {
```
Comment (korikuzma) [inline tests/data/liftover_variants.py:228]: Is that this variant? https://www.ncbi.nlm.nih.gov/clinvar/variation/3035826/?oq=NC_000017.11:50150040:GC:GCGC&m=NM_032595.5(PPP1R9B):c.472_473dup%20(p.Ala159fs)

It might be good to provide references to places we got these variants

**Review by korikuzma — COMMENTED** [review]
@jennifer-bowser Does this close #196? If so, could you link the issue to the PR?

File: tests/data/liftover_variants.py (hunk 1 above)
Comment (jennifer-bowser) [reply tests/data/liftover_variants.py:228]: Ohhh that would've been a good call but I have no idea where I got all the others from at this point 😭 . I can definitely link this one though!

**Review by jsstevenson — APPROVED** [review]

**Review by korikuzma — APPROVED** [review]

**Review by jsstevenson — APPROVED** [review]
